docker/cytoscape/tests/connect_cytoscape.py

```diff
@@ -0,0 +1,242 @@
+from py2cytoscape import cyrest
+from requests.exceptions import ConnectionError
+import time
+import sys
+import sys
+import requests
+import urllib.request as urllib2		
+import json
+from json import JSONDecodeError
+import subprocess
+
+from requests.models import Response
+
+
+##########################################################################################################
+##########################################################################################################
+'''
+The following functions are a unique implementation for hadndling Cytoscape 
+requests and responses
+
+based off:
+    py2cytoscape.cyrest
+    py2cytoscape.base
+
+'''
+
+HOST = 'localhost'
+IP=HOST # temporary
+PORT = 1234
+VERSION = 'v1'
+BASE_URL = 'http://' + HOST + ':' + str(PORT) + '/' + VERSION + '/'
+HEADERS = {'Content-Type': 'application/json'}
+VERBOSE=False
+
+def checkresponse(r,verbose=False):
+    status=r.status_code
+    if 200 <= status < 300:
+        if verbose:
+            print("response status "+status)
+            sys.stdout.flush()
+        res=None
+    else:
+        print(r, r.status_code)
+        sys.stdout.flush()
+        res="error::"+str(r.status_code)
+    return res
+
+def handle_status_codes(x,verbose=False):
```

Comment:
The logic in this function is confusing.  Can you document what it does and the major cases?